src/main/resources/application.yml

```diff
@@ -99,6 +99,7 @@ temporary-certificate:
           fit-width-mm: 35
           fit-height-mm: 45
   explainer-pdf:
+    filename-prefix: "temporary-certificate-explainer-document"
```

Comment:
Possibly `download-filename-prefix` or `response-filename-prefix` (I was confusing it with template filename)

Reply:
Good shout - I'll go with `download-filename-prefix` 👍

Reply:
Done